src/internal.c

```diff
@@ -26983,6 +27086,12 @@ static int HashSkeData(WOLFSSL* ssl, enum wc_HashType hashType,
             ssl->options.sendVerify = SEND_BLANK_CERT;
         }
 
+    #ifdef WOLFSSL_MAXQ10XX_TLS
+        if (ssl->maxq_ctx.device_cert) {
+            ssl->options.sendVerify = SEND_CERT;
```

Comment:
What is this? Is it forcing mutual auth? I don't like it. We have API's to force mutual auth.

Reply:
Removed.